spec/features/facets_spec.rb

```diff
@@ -24,24 +24,24 @@
     before do
       visit '/catalog'
     end
-    it 'card-header in facet-panel-collapse has link with aria-expanded' do
-      a_tag_first = find(:xpath, "//*[@id='facet-panel-collapse']/div[1]/div[1]/a")
+    it 'card-header in facet-panel-collapse has button with aria-expanded' do
```

Comment:
The selector has changed, but the behavior has not.  Updating the spec to use the new selector